test/README.md

```diff
@@ -0,0 +1,274 @@
+<!--
+SPDX-FileCopyrightText: The RamenDR authors
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Ramen test environment
+
+This directory provides tools and configuration for creating Ramen test
+environment.
+
+## Setup
+
+1. Add yourself to the libvirt group (required for minikube kvm2 driver).
+
+   ```
+   sudo usermod -a -G libvirt $(whoami)
+   ```
+
+1. Logout and login again for the change above to be in effect.
+
+1. Install minikube, for example on RHEL/CentOS/Fedora:
+
+   ```
+   sudo dnf install https://storage.googleapis.com/minikube/releases/latest/minikube-latest.x86_64.rpm
+   ```
+
+1. Install the `drenv` package in a virtual environment:
+
+   ```
+   python3 -m venv ~/.venv/ramen
+   source ~/.venv/ramen/bin/activate
+   pip install --upgrade pip
+   pip install -e .
```

Comment:
Need to say that you have to be under `ramen/test` directory

Reply:
Right, will add.

Reply:
Current version explains about changing directories in the next section, and this section
was modified to work from the root directory.